.github/workflows/pr.yml

```
@@ -202,6 +202,13 @@ jobs:
     uses: ./.github/workflows/e2e-atoms.yml
     secrets: inherit
 
+  api-v2-breaking-changes:
+    name: API v2 Breaking Changes
+    needs: [changes]
+    if: ${{ needs.check-label.outputs.run-e2e == 'true' && needs.changes.outputs.has-files-requiring-all-checks == 'true' }}
```

actionlint (1.7.7)

property "check-label" is not defined in object type {changes: {outputs: {commit-sha: string; has-files-requiring-all-checks: string}; result: string}}

(expression)